### PR TITLE
Custom fonts

### PR DESCRIPTION
**Pull request by shadzik**
Add possibility to choose a font.

Similarily to image stickers, if we define a fontChooserContainerView, then we can select a font and then procceed with adding the textSticker.

### CODE REVIEW

File: ZLImageEditor.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -289,7 +289,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (longitachi) [inline ZLImageEditor.xcodeproj/project.pbxproj:292]: Why upgrade the minimum supported version from 9.0 to 12.0?

Comment (shadzik) [reply ZLImageEditor.xcodeproj/project.pbxproj:292]: That was Xcode 13.3.1 and it's recommended settings. I can revert it if you still want to support iOS 9

Comment (longitachi) [reply ZLImageEditor.xcodeproj/project.pbxproj:292]: Well, please revert the change of this line.

Comment (shadzik) [reply ZLImageEditor.xcodeproj/project.pbxproj]: Done

**Pull request merged**